lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/viewmodel/FindMealViewModel.kt

```diff
@@ -1,7 +1,86 @@
 package com.kinandcarta.lib.find.meal.viewmodel
 
+import android.location.Location
+import androidx.annotation.RequiresPermission
+import androidx.hilt.lifecycle.ViewModelInject
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.google.android.gms.location.FusedLocationProviderClient
+import com.google.android.gms.location.LocationCallback
+import com.google.android.gms.location.LocationRequest
+import com.google.android.gms.location.LocationResult
+import com.kcc.kmmhackathon.shared.MealsSDK
+import com.kcc.kmmhackathon.shared.entity.Meal
+import com.kcc.kmmhackathon.shared.utility.DistanceUnit
+import kotlinx.coroutines.launch
 
-class FindMealViewModel : ViewModel() {
-    // TODO: Implement the ViewModel
+typealias Meals = List<Meal>
+
+class FindMealViewModel @ViewModelInject constructor(
```

Comment:
`@ViewModelInject` is a special version of `@Inject` which we'll see more as we start to use Hilt more. Without `ViewModelInject` we need to jump through some hoops to inject directly into the constructor, so Dagger is more annoying, but thankfully they added this in Hilt so we can just do `by viewModels()` in the Fragment